.github/workflows/binary-applications.yml

```diff
@@ -214,17 +221,29 @@ jobs:
               run: make deploy
 
             - name: Upload binaries artifact
-              uses: actions/upload-artifact@v2-preview
+              uses: actions/upload-artifact@v2
               with:
                   name: InVEST-mac-binaries
                   path: dist/InVEST-*-mac.zip
 
             - name: Upload DMG artifact
-              uses: actions/upload-artifact@v2-preview
+              uses: actions/upload-artifact@v2
               with:
                   name: InVEST-dmg
                   path: dist/InVEST*.dmg
 
+            - name: Tar the workspace to preserve permissions

```

Comment:
Out of curiosity, why is this needed to preserve permissions? Is there no equivalent for windows or it's just not needed?

Reply:
> why is this needed to preserve permissions?

Because of [this warning](https://github.com/actions/upload-artifact#permission-loss) in the docs for `actions/upload-artifact`.

> Is there no equivalent for windows or it's just not needed?

Actually, the same thing would almost certainly work for the Windows binaries, it just wasn't clear if it would be needed.  If you'd like I'd be happy to add it in there just in case, but I didn't have need for it for this PR and so I didn't add it.  Let me know what you think!

Reply:
oh, thanks! it's probably fine to leave as is.